Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -1088,7 +1088,10 @@ var Navigator = React.createClass({
     }
     return React.cloneElement(this.props.navigationBar, {
       ref: (navBar) => {
-        this.props.navigationBar.ref instanceof Function && this.props.navigationBar.ref(navBar);
+        var navigationBar = this.props.navigationBar;
+        if (navigationBar && navigationBar.ref instanceof Function) {
```

Comment:
Wondering why `navigationBar.ref instanceof Function` was used instead of `typeof navigationBar.ref === 'function'`